test/cases/ignore-next.out.css

```diff
@@ -0,0 +1,29 @@
+ignore-property {
+  /* autoprefixer: ignore next */
+  mask: none;
+  background: -webkit-linear-gradient(top, white, black);
+  background: -o-linear-gradient(top, white, black);
+  background: linear-gradient(to bottom, white, black);
+}
+
+ignore-property_nospaces-comment {
+  /* autoprefixer:ignore next */
+  mask: none;
+  background: -webkit-linear-gradient(top, white, black);
+  background: -o-linear-gradient(top, white, black);
+  background: linear-gradient(to bottom, white, black);
+}
+
+/* autoprefixer: ignore next */
+ingore-rule {
+  mask: none;
+  background: linear-gradient(to bottom, white, black);
```

Comment:
We need prefixes here and inside next at-rule. This comment should affect only selector and at-rule params on next node.

Or we will need to think about combining `ignore next` and `off/on`.

Reply:
Fixed.  Disable Autoprefixer only for selector, at-rule or property.